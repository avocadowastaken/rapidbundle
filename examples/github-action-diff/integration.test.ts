import { testExample } from "../../test/testExample";

testExample(import.meta.url, { isCI: true });
