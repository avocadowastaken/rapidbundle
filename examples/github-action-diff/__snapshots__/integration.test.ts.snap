// Vitest Snapshot v1

exports[`github-action-diff > dist 1`] = `
[
  "main.js",
]
`;

exports[`github-action-diff > dist/main.js 1`] = `
"use strict";

// src/main.ts
console.log("Hello");

`;

exports[`github-action-diff > logs 1`] = `
✔ Resolving build manifests
  ─ ℹ Checking 'action.yml'
  ─ ℹ Checking 'package.json'
  ✔ Run preparations
  ─ ℹ Removing 'dist' directory
  ✔ Making bundle from 'action.yml'
  ─ ℹ Using '.runs.main' entry: ./src/main.ts
  ─ ℹ Using '.runs.using' entry: node16
  ─ ℹ Checking build difference
------
Exit Code: 0
`;
